crates/autopilot/src/solvable_orders.rs

```diff
@@ -407,14 +376,6 @@ async fn update_task(cache: Weak<SolvableOrdersCache>, current_block: CurrentBlo
     }
 }
 
-#[async_trait::async_trait]
-impl Maintaining for SolvableOrdersCache {
```

Comment:
Removed because we already spawn a maintenance task.

Reply:
I think we shouldn't remove this. We should update solvable orders on new blocks (as well as periodically because of order cancellation).